resource/global/txwy/resource/tasks.json

```diff
@@ -199,9 +199,8 @@
             ["^《", ""],
             ["<《", ""],
             ["^_", ""],
-            ["_-", ""],
             ["&<", ""],
-            ["<-", ""],
+            [".*-", ""],
```

Comment:
This could be pretty dangerous since it's a pretty big wildcard `anything-` -> ` `

Reply:
I don't try this yet, I hope it will be fine. I'll try this tomorrow